src/android/NetworkManager.java

```diff
@@ -147,6 +132,69 @@ public boolean execute(String action, JSONArray args, CallbackContext callbackCo
      * Stop network receiver.
      */
     public void onDestroy() {
+        this.unregisterReceiver();
+    }
+
+    @Override
+    public void onPause(boolean multitasking) {
+        this.unregisterReceiver();
+    }
+
+    @Override
+    public void onResume(boolean multitasking) {
+        super.onResume(multitasking);
+
+        this.unregisterReceiver();
```

Comment:
Why unregister here when it is already done during `onPause()`?

Reply:
I didn't review the old patch, I'll try to identify that with the patch's creator and I'll reopen the PR if needed

Reply:
Given the semantics of the `unregisterReceiver()` method I do not see any harm in having it in both lifecycle callback methods. Was just wondering if it was an oversight or overzealous implementation.